CustomerAPI/Validators/CustomerValidator.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Data.Entities;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Validators
+{
+    public class CustomerValidator : AbstractValidator<Customer>
+    {
+        public CustomerValidator()
+        {
+
+            RuleFor(customer => customer.FullName)
+                .NotEmpty().WithMessage("Name cannot remain empty")
+                .MinimumLength(4);
+
+            RuleFor(customer => customer.Email)
+                .NotEmpty().WithMessage("Email cannot remain empty")
+                .EmailAddress().WithMessage("Email is not valId");
+
+            RuleFor(customer => customer.EmailConfirmation)
+                .NotEmpty().WithMessage("Confirm Email")
+                .Equal(customer => customer.Email).WithMessage("Email confirmation is different than Email");
+
+            RuleFor(customer => customer.Cpf)
+                .NotEmpty().WithMessage("Cpf cannot remain empty")
+                .Must(CpfValIdation);
+
+            RuleFor(customer => customer.Cellphone)
+                .NotEmpty().WithMessage("Cellphone cannot remain empty");
+
+            RuleFor(customer => customer.DateOfBirth)
+                .NotEmpty().WithMessage("Date_of_birth cannot remain empty")
+                .Must(VerifyAge).WithMessage("Customer cannot be registered if he is a minor");
```

Comment:
Validações próprias do FluentValidation não precisa sobrescrever suas mensagem pois são bem descritivas
```suggestion
                .NotEmpty()
                .Must(VerifyAge).WithMessage("'Customer' must be legal age.");
```